frontend/src/components/ui/Nav/AccountNav.jsx

```diff
@@ -0,0 +1,41 @@
+import { Link } from 'react-router-dom'
+import { useAuth } from '../../../context/AuthContext'
+
+import styles from './Nav.module.css'
+
+function AccountNav({onClick}) {
+  const { logout } = useAuth()
+
+  return (
+    <div className={styles.account}>
+      <Link
+        to='/profile'
+        className={styles.acctlink}
+        tabindex='-1'
+        onClick={onClick}
+        >
+        My Profile
+      </Link>
+      <Link
+        to='/messages'
+        className={styles.acctlink}
+        tabindex='-1'
+        onClick={onClick}
+        >
+        Messages
+      </Link>
+      <Link
+        className={styles.acctlink}
+        tabindex='-1'
+        onClick={() => {
+          onClick()
+          logout() 
+        }}
```

Comment:
This is correctly logging the user out but the navigate to /login doesn't work for some reason, I would just add a to='messages' as well

Reply:
Fixed in [f6a47fc](https://github.com/lujorryn/PetPal-G0346/pull/22/commits/f6a47fc78978550def9d1e1674e9d96960f35e95)
Not really sure why navigate doesn't work either. I added `to='/login'` so it should be working now.